WalletLibrary/WalletLibrary/Extensions/Mappings/VCSDK/Issuance/PresentationDescriptor+Mappable.swift

```diff
@@ -31,7 +27,7 @@ extension PresentationDescriptor: Mappable {
                                      types: [credentialType],
                                      purpose: nil,
                                      issuanceOptions: issuanceOptions ?? [],
-                                     id: nil,
+                                     id: credentialType,
```

Comment:
Shouldn't this be a unique ID? As is, someone could request the same credential type with different issuer constraints, and the last would squash the prior.

Reply:
ohh, this is during issuance so there isnt any issuers or constraints. It needs to be the credentialType because that is the mapping our issuance service expects. This will change when we implement issuance v2.